sdk/identity/identity/.gitignore

```diff
@@ -1,3 +1,6 @@
 src/**/*.js
 !assets/fake-cert.pem
 !assets/fake-cert-password.pem
+sshKey
+sshKey.pub
+
```

Comment:
Maybe it would make sense to consolidate these entries? 

1. Remove them from the top-level .gitignore
2. Delete sdk/identity/.gitignore
3. Add the entries from `sdk/identity/.gitignore` here 

What do you think? unless sshKey and sshKey.pub end up at the wrong directory?